### PR TITLE
update from source install instructions

### PR DESCRIPTION
**Pull request by JayceFayne**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -131,10 +131,12 @@ To build Typst yourself, you need to have the [latest stable Rust][rust]
 installed. Then, you can build the CLI with the following command:
 
 ```sh
-cargo build -p typst-cli --release
+git clone https://github.com/typst/typst
```
Comment (AntoniosBarotsis) [inline README.md:134]: Maybe add a `--depth 1` here as there's a decent amount of commits already.

**Comment by AntoniosBarotsis** [comment]
Maybe it's worth waiting for #108 so that we can add that to the readme as well.

**Review by CGMossa — CHANGES_REQUESTED** [review]
If I actually wanted to do this, I'd just:
```
cargo install --git https://github.com/typst/typst
```

**Comment by laurmaedje** [comment]
I added something along these lines to the Readme.

**Pull request closed without merging**